Do not use multiarch on Debian/ubuntu.

From: Tom Rothamel <tom@rothamel.us>

Since everything is a cross-compile, multiarch support has no purpose
and just gets in the way.
---
 setup.py |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/setup.py b/setup.py
index 9a5887b..7b733b0 100644
--- a/setup.py
+++ b/setup.py
@@ -629,6 +629,9 @@ class PyBuildExt(build_ext):
             self.failed.append(ext.name)
 
     def add_multiarch_paths(self):
+        # renpy-build - disable this, since everything is a cross-compile.
+        return
+
         # Debian/Ubuntu multiarch support.
         # https://wiki.ubuntu.com/MultiarchSpec
         cc = sysconfig.get_config_var('CC')
